Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/model/GameFile.kt

```diff
@@ -0,0 +1,68 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+package org.dolphinemu.dolphinemu.model
+
+import androidx.annotation.Keep
+
+class GameFile @Keep private constructor(@Keep private val pointer: Long) {
```

Comment:
Why `@Keep` for the parameter?

Reply:
Yeah, it should only be needed for the constructor itself.

Reply:
Oh, wait. It's because the parameter effectively takes the place of what was an attribute in Java. Assuming this has been tested not to crash, looks fine to me.

Reply:
Oooh. Right.

Reply:
Turns out I can just place `@Keep` on the class itself to cover both